### PR TITLE
Added drush support.

### PR DESCRIPTION
**Pull request by waspper**

### CODE REVIEW

**Comment by andypost** [comment]
Which drush version it supposed to support?

**Comment by waspper** [comment]
Added drupal dependency on composer.json. Also, other D9 checkings. The "drupal-check" script didn't show any issue.

File: composer.json [hunk 1]
```diff
@@ -8,5 +8,15 @@
...
     }
-  ]
+  ],
+  "require": {
+    "drupal/core": "^8.7.7 || ^9"
+  },
+  "extra": {
+    "drush": {
+      "services": {
+        "drush.services.yml": "^9"
```
Comment (andypost) [inline composer.json:18]: it should be `^9 || ^10` because drush 9 soon will be unsupported

Comment (andypost) [reply composer.json:18]: Yep, needs follow-up https://docs.drush.org/en/master/commands/#specifying-the-services-file

**Pull request merged**